app/jobs/regular/emit_web_hook_event.rb

```diff
@@ -23,9 +46,14 @@ def execute(args)
     private
 
     def web_hook_request
-      uri = URI(@web_hook.payload_url)
+      web_hook_event = WebHookEvent.create!(web_hook_id: @web_hook.id)
 
-      conn = Excon.new(uri.to_s, ssl_verify_peer: @web_hook.verify_certificate)
+      uri = URI(@web_hook.payload_url)
+      conn = Excon.new(uri.to_s,
+                       ssl_verify_peer: @web_hook.verify_certificate,
+                       retry_limit: 0,
+                       instrumentor: ExconWebHookEventInstrumentor,
```

Comment:
I'm not really a fan of this implementation as it couples the logs to Excon. If we eventually decide to not use Excon, this portion of the code will break. I rather we check the response and create an entry in the log.